test/e2e/storage/persistent_volumes-local.go

```diff
@@ -265,6 +265,46 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 				})
 			})
 
+			Context("Set fsGroup for local volume", func() {
+
+				It("should enforce fsGroup for one pod", func() {
+					fsGroup := int64(1234)
+					pod, err := createLocalPod(config, testVol, &fsGroup)
+					framework.ExpectNoError(err)
+					By("Checking fsGroup is enforced")
+					_, err = framework.LookForStringInPodExec(config.ns, pod.Name, []string{"ls", "-ld", volumeDir}, strconv.Itoa(int(fsGroup)), time.Second*10)
+					Expect(err).NotTo(HaveOccurred(), "failed: getting the right privileges in the file %v", int(fsGroup))
+				})
+
+				It("should not enforce fsGroup for second pod", func() {
```

Comment:
Some more test cases:
* 2 pods using the same PV one after another (but not simultaneously)
* 2 pods using same PV the same time with same fsgroup

Reply:
Done.